Update readmes (#190)
phoible/dev: raw-data/UPSID/README.md

# UPSID

The `UPSID` folder contains data from the UCLA Phonological Segment 
Inventory Database:

> Maddieson, I., & Precoda, K. (1990). Updating UPSID. _UCLA Working Papers in Phonetics_, 74, 104–111.

The contents and extraction pipeline for these data are described in (chapter 4):

> Moran, Steven. (2012). Phonetics Information Base and Lexicon. PhD thesis, University of Washington. Online: [https://digital.lib.washington.edu/researchworks/handle/1773/22452](https://digital.lib.washington.edu/researchworks/handle/1773/22452).

> The data are available in several files in this directory from the original ASCII dump. These inventories contain phonemes, but **they do not contain tone**.

Reviewer: ```suggestion
The data are available in several files in this directory from the original ASCII dump. These inventories contain only phonemes, with no information on allophones or linguistic tone.
```
